src/main/java/teammates/common/util/Const.java

```diff
@@ -91,39 +91,27 @@
-    public static final Instant INSTANT_REPRESENTS_LATER;
-    public static final Instant INSTANT_REPRESENTS_NOW;
-    public static final Instant INSTANT_REPRESENTS_DEFAULT_TIMESTAMP;
-
-    // TODO: remove and rename Instant constants above to the original names of the Date constants below
-    public static final Date TIME_REPRESENTS_FOLLOW_OPENING;
-    public static final Date TIME_REPRESENTS_FOLLOW_VISIBLE;
-    public static final Date TIME_REPRESENTS_NEVER;
-    public static final Date TIME_REPRESENTS_LATER;
-    public static final Date TIME_REPRESENTS_NOW;
+    public static final Instant TIME_REPRESENTS_FOLLOW_OPENING;
+    public static final Instant TIME_REPRESENTS_FOLLOW_VISIBLE;
+    public static final Instant TIME_REPRESENTS_NEVER;
+    public static final Instant TIME_REPRESENTS_LATER;
+    public static final Instant TIME_REPRESENTS_NOW;
+    public static final Instant TIME_REPRESENTS_DEFAULT_TIMESTAMP;
 
     @Deprecated
     public static final Date TIME_REPRESENTS_DEFAULT_TIMESTAMP_DATE;
 
     public static final String ERROR_FEEDBACK_EMAIL_SUBJECT = "User-submitted Error Report";
 
     static {
-        INSTANT_REPRESENTS_FOLLOW_OPENING = TimeHelper.parseInstant("1970-12-31 12:00 AM +0000");
-        INSTANT_REPRESENTS_FOLLOW_VISIBLE = TimeHelper.parseInstant("1970-06-22 12:00 AM +0000");
-        INSTANT_REPRESENTS_NEVER = TimeHelper.parseInstant("1970-11-27 12:00 AM +0000");
-        INSTANT_REPRESENTS_LATER = TimeHelper.parseInstant("1970-01-01 12:00 AM +0000");
-        INSTANT_REPRESENTS_NOW = TimeHelper.parseInstant("1970-02-14 12:00 AM +0000");
-        INSTANT_REPRESENTS_DEFAULT_TIMESTAMP = TimeHelper.parseInstant("2011-01-01 12:00 AM +0000");
-
-        TIME_REPRESENTS_FOLLOW_OPENING = TimeHelper.convertInstantToDate(INSTANT_REPRESENTS_FOLLOW_OPENING);
-        TIME_REPRESENTS_FOLLOW_VISIBLE = TimeHelper.convertInstantToDate(INSTANT_REPRESENTS_FOLLOW_VISIBLE);
-        TIME_REPRESENTS_NEVER = TimeHelper.convertInstantToDate(INSTANT_REPRESENTS_NEVER);
-        TIME_REPRESENTS_LATER = TimeHelper.convertInstantToDate(INSTANT_REPRESENTS_LATER);
-        TIME_REPRESENTS_NOW = TimeHelper.convertInstantToDate(INSTANT_REPRESENTS_NOW);
-        TIME_REPRESENTS_DEFAULT_TIMESTAMP_DATE = TimeHelper.convertInstantToDate(INSTANT_REPRESENTS_DEFAULT_TIMESTAMP);
+        TIME_REPRESENTS_FOLLOW_OPENING = TimeHelper.parseInstant("1970-12-31 12:00 AM +0000");
+        TIME_REPRESENTS_FOLLOW_VISIBLE = TimeHelper.parseInstant("1970-06-22 12:00 AM +0000");
+        TIME_REPRESENTS_NEVER = TimeHelper.parseInstant("1970-11-27 12:00 AM +0000");
+        TIME_REPRESENTS_LATER = TimeHelper.parseInstant("1970-01-01 12:00 AM +0000");
+        TIME_REPRESENTS_NOW = TimeHelper.parseInstant("1970-02-14 12:00 AM +0000");
+        TIME_REPRESENTS_DEFAULT_TIMESTAMP = TimeHelper.parseInstant("2011-01-01 12:00 AM +0000");
+
+        TIME_REPRESENTS_DEFAULT_TIMESTAMP_DATE = TimeHelper.convertInstantToDate(TIME_REPRESENTS_DEFAULT_TIMESTAMP);
```

Comment:
Mark `Deprecated`?

Reply:
The field is marked deprecated a few lines up, during declaration.